README.md

```diff
@@ -29,9 +29,9 @@ If there's no link, it's not done yet! [why not contribute?](contributing.md)
 8. [Dependency Injection](dependency-injection.md) - Learn about dependency injection, how it relates to using interfaces and a primer on io.
 9. [Mocking](mocking.md) - Take some existing untested code and use DI with mocking to test it.
 10. [Concurrency](concurrency.md) - Learn how to write concurrent code to make your software faster.
-11. [Select](select.md) - Learn how to synchronise asynchronous processes elegantly.
+11. [Select](select.md) - Learn how to synchronize asynchronous processes elegantly.
```

Comment:
That's an Americanism, I'd rather it be British-English :)

Reply:
Sorry, I didn't notice that in Grammarly. Now I've changed it back to synchronise.